runtime/oti/j9nonbuilder.h

```diff
@@ -5717,10 +5717,11 @@ typedef struct J9CInterpreterStackFrame {
 	UDATA jitGPRs[32]; /* x0-x31 */
 	U_8 jitFPRs[32 * 8]; /* v0-v31 */
 #elif defined(J9VM_ARCH_RISCV) /* J9VM_ARCH_ARM */
-	UDATA preservedGPRs[13]; /* x2, x8, x9, and x18-x27  */
+	UDATA preservedGPRs[11]; /* x2, x8, x9, and x18-x25  */
```

Comment:
@janvrany Does this have implications for DDR code (two fewer preserved registers)?

Reply:
@keithc-ca To be honest, I do not know. I changed that because in (future) private (JIT) linkage I have chosen `s10` (that is, `x26`) as "vm thread" and `s11` (that is, `x27`) as "java SP".

Reply:
@keithc-ca see https://github.com/janvrany/openj9/blob/jv/riscv-devel/runtime/compiler/riscv/codegen/RVPrivateLinkage.cpp#L94-L95

Reply:
I expect some part of DDR will need to be informed of these decisions, but I'm not sure exactly where. Please keep that in mind as you make similar choices. Thanks.